src/python/pants/backend/python/lint/pylint/subsystem.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.backend.python.subsystems.python_tool_base import PythonToolBase
+from pants.option.custom_types import file_option, shell_str
+
+
+class Pylint(PythonToolBase):
+  options_scope = 'pylint'
+  default_version = 'pylint'
+  default_extra_requirements = ['setuptools']
```

Comment:
Do you know if this is required for Pylint to work properly? Test to figure this out: remove it, run ./pants lint2, and see if Pex prints a warning about Pex using its vendored pkg_resources.

Reply:
Good catch, turns out it's not needed.

Reply:
Awesome! Would be good to check the same with bandit.

Reply:
Sure - it is indeed needed with Bandit.